samples/revoke_dataset_access.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2019 Google LLC
```

Comment:
AFAIK copyright year in the license header for new files should be set to the current year, which would be 2021?

Applies to both files.

Reply:
+1

Reply:
done!

Reply:
done